tools/accuracy_checker/accuracy_checker/annotation_converters/_nlp_common.py

```diff
@@ -53,11 +61,146 @@ def get_tokenizer(config, lower_case):
     if tokenizer is None:
         raise ConfigError(
             'tokenization parameters is not found, please provide: \n'
-            'for WordPiece tokenization - {}\nfor SentencePiece tokenization - {}\n'.format(
-                ', '.join(WORD_PIECE_PARAMETERS), ', '.join(SENTENCE_PIECE_PARAMETERS))
+            'for WordPiece tokenization - {}\nfor extended WordPiece tokenization - {}\nfor SentencePiece tokenization - {}\n'.format(
+                ', '.join(WORD_PIECE_PARAMETERS), ', '.join(EXTENDED_WORD_PIECE_PARAMETERS), ', '.join(SENTENCE_PIECE_PARAMETERS))
         )
     return tokenizer
 
+class BasicTokenizer(object):
```

Comment:
How much code does this class share with the `WordPieceTokenizer` class below? `BasicTokenizer._tokenize_chinese_chars` is an exact copy of `WordPieceTokenizer._tokenize_chinese_chars`. Can we eliminate code duplication?